apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/stu3/providers/DMEClaimTransformerTest.java

```diff
@@ -47,13 +46,13 @@ public void transformSampleARecord() throws FHIRException, IOException {
             .get();
 
     ExplanationOfBenefit eob =
-        DMEClaimTransformer.transform(
-            new TransformerContext(
-                new MetricRegistry(),
-                Optional.of(true),
-                FdaDrugCodeDisplayLookup.createDrugCodeLookupForTesting(),
-                new NPIOrgLookup()),
-            claim);
+        TransformerTestUtils.transformRifRecordToEob(
+            claim,
```

Comment:
Why are we calling this helper method instead of just calling the actual transformer?

Reply:
Not sure why I did this...it may have been a vestige of copy-paste from some other test. Will remove.

Reply:
in digging into this more, this is stu3 code/tests. that is what was there....I didn't add this.

Reply:
invoking transform method within unit tests.